internal/sirius/get_firm_details.go

```diff
@@ -45,5 +53,17 @@ func (c *Client) GetFirmDetails(ctx Context, firmId int) (FirmDetails, error) {
 	}
 
 	err = json.NewDecoder(resp.Body).Decode(&v)
+
+	v.TotalNumberOfDeputies = calculateNumberOfDeputies(v.Deputies)
```

Comment:
This can just be `len(v.Deputies)`. I'd probably also rename the variable to something less verbose, like `DeputyCount`, though that's up to you if you think the current wording is easier to parse.

In fact, you don't event really need this as a variable at all if you are only calculating it for use in the template. Go templates come with built in functions, one of which is `len`. So in your template, you can just write `{{ len .FirmDetails.Deputies }}` and it will calculate it for you.